projects/models.py

```diff
@@ -333,12 +317,36 @@ class PublicationSource(models.Model):
         (GOOGLE_SCHOLAR, "Google Scholar"),
     ]
 
+    APPROVED_WITH_PUBLICATION = "publication"
+    APPROVED_WITH_JUSTIFICATION = "justification"
+    APPROVED_WITH_EMAIL = "email"
+    APPROVED_WITH_PENDING_REVIEW = "pending_review"
+
+    APPROVED_WITH = [
+        (APPROVED_WITH_PUBLICATION, "Publication"),
+        (APPROVED_WITH_JUSTIFICATION, "Justification"),
+        (APPROVED_WITH_EMAIL, "Email"),
+        (APPROVED_WITH_PENDING_REVIEW, "Review Pending")
+    ]
+
+    SOURCE_REPORT_FIELDS = [
+        "name",
+        "is_found_by_algorithm",
+        "is_cited",
+        "is_acknowledged",
+        "approved_with"
+    ]
+
     publication = models.ForeignKey(Publication, related_name="sources", on_delete=models.CASCADE)
     name = models.CharField(max_length=30, choices=SOURCES)
     citation_count = models.IntegerField(default=0, null=False)
+    entry_created_date = models.DateField(auto_now_add=True)
     # if the publication identified by the source
-    # using our algorithm to find publications ref Chamaleon
-    found_by_algorithm = models.BooleanField(default=False, null=False)
+    # using our algorithm to find publications ref Chamaeleon
+    is_found_by_algorithm = models.BooleanField(default=False, null=False)
+    is_cited = models.BooleanField(default=False, null=False)
```

Comment:
Nit: the name is a bit confusing. I think `cites_chameleon` and similarly `acknowledges_chameleon` for the next field would be more clear.